xml/depl_conf_admin_crowbar.xml

```diff
@@ -109,14 +109,14 @@
     linuxbridge, openvswitch plus VLAN, or cisco_nexus). In this case, you
     need to specify a correct <guimenu>VLAN ID</guimenu> for this network.
    </para>
-   <para>
+<para>
     <emphasis role="bold">nova-floating</emphasis>: When using a VLAN for
     <literal>nova-floating</literal> (which is the default), the <guimenu>USE
     VLAN</guimenu> and <guimenu>VLAN ID</guimenu> settings for
-    <guimenu>nova-floating</guimenu> and <guimenu>public</guimenu> need to be
-    the same. When not using a VLAN for <literal>nova-floating</literal>, it
-    needs to use a different physical network interface than the
-    <literal>nova_fixed</literal> network.
+    <guimenu>nova-floating</guimenu> and <guimenu>public</guimenu> default to
+    the same.
+    </para>
+    <para>You have the option of separating public and floating networks with a custom configuration. Configure your own separate floating network (not as a subnet of the public network), and give the floating network its own router. For example, define <literal>nova-floating</literal> as part of an external network with a custom <literal>bridge-name</literal>. When you are using different networks and OpenVSwitch is configured, the pre-defined <literal>bridge-name</literal> won't work.
```

Comment:
please use 80-character lines and use 1-space indentation. (you can usually use `daps-xmlformat -i [file]` to pretty much get to the style we want -- do make sure to use daps 2.3 or newer before, some older daps versions unfortunately wrote error messages into the file)